doc/sphinx/source/sycl_dpcpp.rst

```diff
@@ -0,0 +1,197 @@
+==========================================
+Using PoCL as the OpenCL backend for DPC++
+==========================================
+
+SYCL is programming model that enables single-source C++ development for
+heterogenous computing. Compared to OpenCL, SYCL operates at a higher level
+of abstraction, relying on lower-level implementations like OpenCL for device
+offloading.
+
+If the OpenCL backend is used, the SYCL runtime translates the SYCL API calls
```

Comment:
Check when writing about DPC++ and SYCL. A runtime might or might not support OpenCL/SPIR-V. Initially it was a must, but they diverged in this front and it's not required to be a layer on OpenCL.